scripts/exregional_run_fcst.sh

```diff
@@ -9,6 +9,7 @@
 #
 . ${GLOBAL_VAR_DEFNS_FP}
 . $USHDIR/source_util_funcs.sh
+. $USHDIR/set_FV3nml_stoch_params.sh
```

Comment:
@guoqing-noaa @chunhuazhou The set_FV3nml_stoch_params.sh is made available even for deterministic runs in case stochastic tests are required for single-member runs.  I would leave this in here for that purpose and since this is how the authoritative workflow is set up.

Reply:
Thanks for the clarification! @JeffBeck-NOAA This call is moved from generate__FV3LAM_wflow.sh so that it can be applied for forecast jobs for each member.